modules/player.py

```diff
@@ -0,0 +1,35 @@
+"""The Player class, which contains the hand, stash and status of one player.
+
+"""
+
+from hand import Hand
+from stash import Stash
+
+
+class Player:
```

Comment:
These classes look great! Even though these modules only contain just the classes now, I would duplicate the module docstrings as each of the class docstrings, then when you start adding in helper functions the module docstring can change.